pkg/machines/components/vmOverviewTabLibvirt.jsx

```diff
@@ -85,19 +86,13 @@ class VmOverviewTabLibvirt extends React.Component {
 
     componentDidMount() {
         this._isMounted = true;
-        getDomainCapabilities(this.props.vm.connectionName)
+        getDomainCapabilities(this.props.vm.connectionName, this.props.vm.arch, this.props.vm.emulatedMachine)
```

Comment:
No, but that could come in other PR. It's not a problem that this will be merged even if you cannot change architecture and emulated machine in the UI, since it still fixes the problem where we use incorrect value for max vcpu (this can happen even in VMs where default architecture and emulated machines are used).

Reply:
ok got it